build.gradle

```diff
@@ -18,7 +18,7 @@ sourceCompatibility = JavaVersion.VERSION_11
 targetCompatibility = JavaVersion.VERSION_11
 
 ext {
-    grpcVersion         = '1.32.1'
+    grpcVersion         = '1.48.1'
```

Comment:
Just noticed that it is not used anywhere. Maybe we should remove it? The same goes for other properties that are not used

Reply:
it used here: artifact = "io.grpc:protoc-gen-grpc-java:${grpcVersion}"

Reply:
but I removed slfj4_version and kotlin_version

Reply:
> it used here: artifact = "io.grpc:protoc-gen-grpc-java:${grpcVersion}"

Sorry, missed that.